### PR TITLE
Protected APIs marked with `@internal` for SharedMap and Counter

### PR DESCRIPTION
**Pull request by skylerjokiel**
Adding @internal prevents APIs from being on the public docs.

### CODE REVIEW

File: api-report/map.api.md [hunk 1]
```diff
@@ -169,8 +170,14 @@ export class SharedDirectory extends SharedObject<ISharedDirectoryEvents> implem
     keys(): IterableIterator<string>;
     // (undocumented)
     protected loadCore(storage: IChannelStorageService): Promise<void>;
+    // Warning: (ae-forgotten-export) The symbol "LocalValueMaker" needs to be exported by the entry point index.d.ts
+    //
```
Comment (tylerbutler) [inline api-report/map.api.md:174]: You need to export this type too.

File: api-report/map.api.md [hunk 2]
```diff
@@ -182,6 +189,10 @@ export class SharedDirectory extends SharedObject<ISharedDirectoryEvents> implem
     // (undocumented)
     protected snapshotCore(serializer: IFluidSerializer): ITree;
     subdirectories(): IterableIterator<[string, IDirectory]>;
+    // Warning: (ae-forgotten-export) The symbol "IDirectoryOperation" needs to be exported by the entry point index.d.ts
+    //
```
Comment (tylerbutler) [inline api-report/map.api.md:193]: Same here.

**Review by tylerbutler — APPROVED** [review]
Address the missing exports and ship it. :)

**Review by sumedhb1995 — APPROVED** [review]

**Comment by sumedhb1995** [comment]
Build blocker is probably being caused by the package-lock and lerna-pacakge-lock changes. Leaving those out of the PR should allow it to pass

**Comment by curtisman** [comment]
>   "lockfileVersion": 2,

are we forcing npm 7?

---
Refers to: package-lock.json:4 in ce6320e. [](commit_id = ce6320e043dbecafc7b0451bbb0569163087a903, deletion_comment = False)

**Pull request merged**